fil-proofs-tooling/src/bin/benchy/prodbench.rs

```diff
@@ -167,10 +166,10 @@ fn configure_global_config(inputs: &ProdbenchInputs) {
         .write()
         .expect("POREP_PARTITIONS poisoned")
         .insert(inputs.sector_size_bytes(), inputs.porep_partitions);
-    POREP_MINIMUM_CHALLENGES
-        .write()
-        .expect("POREP_MINIMUM_CHALLENGES poisoned")
-        .insert(inputs.sector_size_bytes(), inputs.porep_challenges);
+    // POREP_MINIMUM_CHALLENGES
```

Comment:
I think that it still makes sense to allow `benchy` to configure the number of PoRep challenges, as `benchy` is used to benchmark varrying PoRep configurations, not just the the default/production configurations.